modules/custom/az_news/az_news_marketing_cloud/templates/field--node--title.html.twig

```diff
@@ -0,0 +1 @@
+{{ elements.smart_title }}
```

Comment:
This seems like it might be overriding the output for all node titles rather than only ones that are displayed via marketing cloud output.